.gitignore

```diff
@@ -50,3 +50,4 @@ install_manifest.txt
 !/.buildkite/Makefile
 .vscode
 .metadata
+/wrapper
```

Comment:
Consider renaming to something with "maven" or "mvn" in its name, unless this is a shared directory for all kinds of wrappers.